### ISSUE TITLE
Pizza Tower Switch Port [PizzaTowerSwitch]

### ISSUE THREAD

**Issue by cpm761** [Rinnegatamante/YoYo-Loader-Vita-Compatibility#1078]
NOTE: IF YOU'RE SUBMITTING A REPORT FOR A GAMING HAVING ISSUES, PLEASE PERFORM A RUN WITH DEBUG MODE ON AND POST THE RELATED LOG FILE HERE!

# Game summary
- Game name: Pizza Tower Switch homebrew port (https://github.com/TurtleHouse/PizzaTowerSwitch-FMOD)
- Game version: 2023.2
- Asset swap with Undead Survivor

# YoYo Loader summary
- Version: f8b628

# Issues
Game shows loading bar but crashes when finished. Log shows errors related to memory allocation so the game might be too large?

# Recommended labels
Crash

[yyl.log](https://github.com/Rinnegatamante/YoYo-Loader-Vita-Compatibility/files/11931591/yyl.log)

**Comment by HunterBoy344** [comment]
There seems to be a lot going wrong in that log. Failed memory allocations, missing files, failed texture loads…

Ignoring all that, this is almost certainly a hardware issue. The Vita has just 512MB of RAM and a 32 bit processor, compared to the Switch’s 4GB of RAM and 64 bit processor. According to the Steam page for Pizza Tower, a 64 bit processor is required to run the game, and [this discussion explains why:](https://steamcommunity.com/app/2231450/discussions/0/3759978783504250397/) Pizza Tower was made in a version of GameMaker/GMS2 newer than version 2022.8, which discontinued support for 32 bit Windows exports. No idea whether this also applies to the unofficial Switch port, but I would assume it does given that the Switch is also a 64 bit system.

Even if that’s not actually the issue and this IS possible to run in any capacity, wouldn’t memory be a problem? The Vita has the absolute bare minimum amount of RAM necessary to run Pizza Tower, and I bet some of that is already dedicated to running the OS.

I hate to crush your dreams but this seems impossible.

**Comment by FreindlyNeighborhoodBeetle** [comment]
i agree with the memory thing but you are wrong about the 64 bit thing. undertale yellow runs (without sound but i think thats just cuz of how they handled the files)
